app/views/support_interface/countries/index.html.erb

```diff
@@ -16,11 +16,14 @@
               <%= govuk_tag(text: "Ineligible", colour: "red", classes: ["govuk-!-margin-left-2"]) %>
             <% end %>
 
-            <% if country.eligibility_skip_questions %>
-              <%= govuk_tag(text: "Reduced eligibility", colour: "yellow", classes: ["govuk-!-margin-left-2"]) %>
-            <% else %>
-              <%= govuk_tag(text: "Standard eligibility", colour: "grey", classes: ["govuk-!-margin-left-2"]) %>
-            <% end %>
+            <% if country.subject_limited %>
+  <%= govuk_tag(text: "Subject limited eligibility", colour: "yellow", classes: ["govuk-!-margin-left-2"]) %>
+<% elsif country.eligibility_skip_questions %>
+  <%= govuk_tag(text: "Reduced eligibility", colour: "yellow", classes: ["govuk-!-margin-left-2"]) %>
+<% else %>
+  <%= govuk_tag(text: "Standard eligibility", colour: "grey", classes: ["govuk-!-margin-left-2"]) %>
+<% end %>
```

Comment:
```suggestion
            <% if country.subject_limited %>
              <%= govuk_tag(text: "Expanded eligibility", colour: "yellow", classes: ["govuk-!-margin-left-2"]) %>
            <% elsif country.eligibility_skip_questions %>
              <%= govuk_tag(text: "Reduced eligibility", colour: "yellow", classes: ["govuk-!-margin-left-2"]) %>
            <% else %>
              <%= govuk_tag(text: "Standard eligibility", colour: "grey", classes: ["govuk-!-margin-left-2"]) %>
            <% end %>
```

I think the indentation is wrong here, but also it occurs to me that we should probably match the naming of the question.